src/BloomExe/Workspace/BloomClipboard.cs

```diff
@@ -88,15 +118,29 @@ public static PalasoImage GetImageFromClipboard()
 
 			return null;
 #else
-			if (Clipboard.ContainsImage())
-			{
-				return PalasoImage.FromImage(Clipboard.GetImage());
-			}
-
 			var dataObject = Clipboard.GetDataObject();
 			if (dataObject == null)
 				return null;
 
+			var formatsAvailable = dataObject.GetFormats();
+			PalasoImage image = null;
+			var textData = String.Empty;
+			if (formatsAvailable.Contains(DataFormats.UnicodeText))
+				textData = dataObject.GetData(DataFormats.UnicodeText) as String;
+			if (Clipboard.ContainsImage())
+			{
+				image = PalasoImage.FromImage(Clipboard.GetImage());
+				if (String.IsNullOrEmpty(textData))
+				{
+					return image;
```

Comment:
good point. I've reworked this bit.
Also the 'formatsAvailable' was a bit of a red herring. Now just using GetDataPresent().